README.md

```diff
@@ -13,17 +14,54 @@ If you want to use it directly in a browser use a CDN like [Skypack](https://www
 ```javascript
 import urlJoin from 'url-join';
 
-const fullUrl = urlJoin('http://www.google.com', 'a', '/b/cd', '?foo=123');
+const id = '42'
+const postUrl = urlJoin('/posts', id, '/metadata/');
+// => "/posts/42/metadata"
+```
+## Use the Web Standard APIs for Other Cases
 
-console.log(fullUrl);
+Web Sandard APIs [URL][1] and [URLSearchParams][1] capable of URL operations except manipulating [URL pathnames][2].
+```
+Use this library to manipulate URL pathnames
+                                      👇
+   https  :// www.example.com /posts/42/metadata ?filter=author&page=1 #heading-1
+|-scheme-|    |---domain----| |----pathname----| |------search-------| |--hash--|
 ```
 
-Prints:
+#### [`URL()`][1]
```

Comment:
Let's limit the examples here to a minimal that matches the existing example for `url-join`. I think the MDN documentation is comprehensive enough for users to find their way.

Something like:

```js
const fullUrl = new URL('http://www.google.com');

fullUrl.pathname = '/a/b/c';
fullUrl.searchParams.append('foo', '123');
fullUrl.searchParams.append('bar', '456');
fullUrl.hash = 'heading-1';

fullUrl.toString(); // http://www.google.com/a/b/c?foo=123&bar=456#heading-1
```